docker/Dockerfile

```
@@ -49,7 +49,7 @@ RUN apt-get update && \
         && \
     rm -rf /var/lib/apt/lists/*
 RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
-COPY --from=builder-echidna /root/.local/bin/echidna-test /usr/local/bin/echidna-test
+COPY --from=builder-echidna /root/.local/bin/echidna /usr/local/bin/echidna
```

I think it will break, as the action invokes `echidna-test` in the docker container. Maybe we can provide a symlink here to maintain compatibility
```suggestion
COPY --from=builder-echidna /root/.local/bin/echidna /usr/local/bin/echidna
RUN ln -s /usr/local/bin/echidna /usr/local/bin/echidna-test
```

---

(alternatively we can update the action to check both binary names and choose the one present)